dimod/binary_quadratic_model.py

```diff
@@ -222,38 +211,27 @@ def spin(self):
 
     @property
     def binary(self):
-        """:class:`.BinaryQuadraticModel`: The binary-valued version of the binary quadratic model.
-
-        This property allows the user to access the biases for the appropriate vartype without
-        needing to check the given binary quadratic model.
-
-        Examples:
-            Create a binary-valued binary quadratic model. In this case the :attr:`.binary` attribute
-            refers back to itself.
-
-            >>> bqm = dimod.BinaryQuadraticModel({'a': .5, 'b': .5}, {('a', 'b'): -1}, 0.0, dimod.BINARY)
-            >>> bqm.binary is bqm
-            True
-
-            For a spin-valued binary quadratic model, it's binary-valued counterpart will be built
-            the first time the :attr:`.binary` property is accessed and subsequent reads will use
-            it.
-
-            >>> bqm = dimod.BinaryQuadraticModel({'a': 0, 'b': 0}, {('a', 'b'): -1}, 0.0, dimod.SPIN)
-            >>> bqm.binary.linear
-            {'a': 2.0, 'b': 2.0}
-            >>> bqm.binary.quadratic
-            {('a', 'b'): -4.0}
-            >>> bqm.binary.offset
-            -1.0
-
-            The energy will correspond.
-
-            >>> bqm = dimod.BinaryQuadraticModel({'a': 0, 'b': 0}, {('a', 'b'): -1}, 0.0, dimod.SPIN)
-            >>> bqm.binary.energy({'a': 0, 'b': 1})
-            1.0
-            >>> bqm.energy({'a': -1, 'b': +1})
-            1.0
+        """:class:`.BinaryQuadraticModel`: An instance of the QUBO model subclass
+        of the :class:`.BinaryQuadraticModel` superclass, corresponding to
+        a binary quadratic model with binary variables.
```

Comment:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

In most cases the summary line should fit on a single line

Reply:
@arcondello, this is the one that's more like an argument. The rest looks okay, so should be ready too merge. Thanks!